build.gradle

```
@@ -448,12 +455,18 @@ task testConnected(type: Test) {
     testLogging {
         events = ["passed", "failed", "skipped", "started", "standard_out"]
     }
-    if (isCiServer) {
-        retry {
-          maxRetries = 3
-          maxFailures = 15
+
+    retry {
+        if (isCiServer) {
+            maxRetries = 3
+            maxFailures = 15
+        }
+        else {
```

I'm pretty sure we don't want any retries when running locally by default, but in case you keep the `else`, make sure you use K&R style braces
```suggestion
        } else {
```